setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
This works only on systems that follow the FHS 3.0 standard and e.g. on NixOS it's broken with:

```console
$ /bin/bash
bash: /bin/bash: No such file or directory
```

Reply:
`#!/usr/bin/env bash` would be "universally POSIX way" :-) Should work also in NixOS.